mappings/net/minecraft/client/network/SrvResolver.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_6371 net/minecraft/client/network/SrvResolver
```

Comment:
Should be a redirect resolver (from the non-obf lambda name). Srv is an implementation detail